src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -132,6 +132,7 @@ def create(input_dataset, target, feature=None, validation_set='auto',
     from .._mxnet import _mxnet_utils
     
     start_time = _time.time()
+    accepted_values_for_warm_start = ["auto", "quickdraw_245_v0", None]
```

Comment:
I'm surprised that `None` is an accepted value. I'm guessing this means no warm start, i.e. trained from scratch. That's a cool feature. The doctoring should mention this.

Reply:
Done. Thanks for pointing the missing docs out!